tools/gulptasks/dashboards/cypress.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) Highsoft AS
+ */
+
+const gulp = require('gulp');
+const path = require('path');
+
+/* *
+ *
+ *  Tasks
+ *
+ * */
+
+/**
+ * Test TypeScript declarations in the code folder using tsconfig.json.
+ *
+ * @return {Promise<void>}
+ *         Promise to keep
+ */
+async function testDTS() {
+
+    const processLib = require('../lib/process');
+    const logLib = require('../lib/log');
+
+    await processLib.exec(
+        'npx tsc -p ' +
```

Comment:
Did you forget a commit? Looks a bit too similar to the test-dts task 🤔

Reply:
:(